### PR TITLE
Pyluach

### PR DESCRIPTION
**Pull request by Matt-BF**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Matt-BF** [comment]
@conda-forge/help-python I believe this is all set!

File: recipes/pyluach/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+test:
+  imports:
+    - pyluach
+
+about:
+  home: https://github.com/simlist/pyluach
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/pyluach/meta.yaml]: Please, change it to 
```
license_file: LICENSE.txt
```

Comment (marcelotrevisani) [reply recipes/pyluach/meta.yaml]: And could you please raise an issue on this project to also add the license with the package?

File: recipes/pyluach/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4d4ad7a680390d5fe29bf5191939b801fd1b72a85f5c6713038160a265af62f7
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/pyluach/meta.yaml]: Please, change it to
```
{{ PYTHON }} -m pip install . -vv
```

File: recipes/pyluach/meta.yaml (hunk 1 above)
Comment (Matt-BF) [reply recipes/pyluach/meta.yaml]: Sure thing! Here is the issue #8556

**Pull request merged**